src/v/archival/ntp_archiver_service.cc

```diff
@@ -223,7 +226,10 @@ ss::future<cloud_storage::download_result> ntp_archiver::sync_manifest() {
             vlog(
               _rtclog.debug,
               "Updating the archival_meta_stm in read-replica mode");
-            auto deadline = ss::lowres_clock::now() + _manifest_upload_timeout;
+            auto sync_timeout
+              = config::shard_local_cfg()
+                  .cloud_storage_metadata_sync_timeout_ms.value();
+            auto deadline = ss::lowres_clock::now() + sync_timeout;
```

Comment:
we have `cloud_storage_readreplica_manifest_sync_timeout_ms`, maybe we should use it here?

Reply:
this one for read replicas (as the name suggests)
the `cloud_storage_metadata_sync_timeout_ms` is a timeout for replication and state machine sync, it requires dfferent timeout value